src/mpid/ch4/netmod/ucx/ucx_am_send.h

```diff
@@ -13,6 +13,44 @@
 
 #include "ucx_impl.h"
 
+MPL_STATIC_INLINE_PREFIX int MPIDI_NM_send_coll(const void *buf, MPI_Aint count,
```

Comment:
Check #3880 because I added these functions already, but your implementation might fix the fact that it doesn't catch the error as expected.